src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -161,6 +161,7 @@ public IHttpActionResult Log(IEnumerable<HostLogEntry> logEntries)
                     {
                         traceEvent.Properties.Add(ScriptConstants.TracePropertyFunctionNameKey, logEntry.FunctionName);
                     }
+                    traceEvent.Properties.Add(ScriptConstants.TracePropertyScriptHostInstanceIdKey, _scriptHostManager.Instance.ScriptHostInstanceId);
```

Comment:
Logs to this endpoint are coming from ScaleController for example, so logging a host ID doesn't really make sense for these, since the logs aren't related to or originating from the active host.